dev-server/ui-playground/shared/router.js

```diff
@@ -1,11 +1,11 @@
 import { useEffect, useMemo, useState } from 'preact/hooks';
 
-function routeFromCurrentUrl(baseUrl) {
-  return location.pathname.slice(baseUrl.length);
+function routeFromCurrentUrl(baseURL) {
```

Comment:
Changes here just to casing for convention matching.

Reply:
Do you want to change the function name (`routeFromCurrentUrl`) while you're at it?